cmd/kubeadm/app/phases/upgrade/health.go

```diff
@@ -106,16 +107,25 @@ func apiServerHealthy(client clientset.Interface) error {
 	return nil
 }
 
-// nodesHealthy checks whether all Nodes in the cluster are in the Running state
-func nodesHealthy(client clientset.Interface) error {
```

Comment:
I would actually keep this check and produce warning in case some nodes are not in healthy state.

Reply:
Agree if possible to plumb through a warning -- users may want a heads up if their cluster isn't at full strength.

Reply:
I disagree in principle: kubeadm is not a monitoring service - if the admin wants to be informed that they have a node down, they shouldn't run kubeadm to discover that.